models/factor.go

```diff
@@ -108,9 +121,20 @@ func findFactor(tx *storage.Connection, query string, args ...interface{}) (*Fac
 	return obj, nil
 }
 
+func FindUnverifiedFactorsByUser(tx *storage.Connection, user *User) ([]*Factor, error) {
+	factors := []*Factor{}
+	if err := tx.Q().Where("user_id = ? AND status = ?", user.ID, FactorStateUnverified.String()).All(&factors); err != nil {
+		if errors.Cause(err) == sql.ErrNoRows {
+			return factors, nil
+		}
+		return nil, errors.Wrap(err, "Database error when finding verified MFA factors")
+	}
+	return factors, nil
+}
+
```

Comment:
As mentioned above, if the factors are not necessary to be used in Go code, just delete them.